src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "./IRegistryCoordinator.sol";
+import "./IStakeRegistry.sol";
+import "./IBLSPubkeyRegistry.sol";
+import "./IIndexRegistry.sol";
+
+/**
+ * @title Minimal interface for the `IBLSStakeRegistryCoordinator` contract.
+ * @author Layr Labs, Inc.
+ */
+interface IBLSRegistryCoordinatorWithIndices is IRegistryCoordinator {
+    // STRUCTS
+
+    struct QuorumBitmapUpdate {
+        uint32 updateBlockNumber;
+        uint32 nextUpdateBlockNumber;
+        uint192 quorumBitmap;
+    }
+
+    struct OperatorSetParam {
+        uint32 maxOperatorCount;
+        uint8 kickPercentageOfOperatorStake;
+        uint8 kickPercentageOfAverageStake;
+        uint8 kickPercentageOfTotalStake;
```

Comment:
the meanings of these parameters need to be way clearer. at minimum should have better documentation.
maybe we can rename them to be clearer too? not too sure on naming though.

Reply:
added comment